dictcc.el

```diff
@@ -61,6 +61,37 @@
   :type 'string
   :group 'dictcc)
 
+(defcustom dictcc-source-languages-alist '(("English" . "en")
```

Comment:
Now that we have source and destination selection, this should be called something more general `dictcc-languages-alist`.